source/develop/release-management/features/storage/qcow2v3.html.md

```diff
@@ -0,0 +1,73 @@
+---
+title: qcow2v3 image format
+category: feature
+authors: mlipchuk
+feature_name: qcow2v3 image format
+feature_modules: engine,vdsm
+feature_status: Implementation
+---
+
+# QCOW2V3 Image Format
+
+
+## Overview
+
+The QCOW image format includes a version number to allow introducing new features that change the image format in an incompatible way.
+Newer QEMU versions support a new format of QCow2 version 3, which is not backwards compatible and introduce
+many improvements like zero clusters, and performance improvement.
+That is the reason we enabled this new compatibility version for QCOW volumes in oVirt.
+
+## Owner
+
+- Name: [Maor Lipchuk](https://github.com/maorlipchuk)
+- Email: <mlipchuk@redhat.com>
+
+
+## Detailed Description
+
+The QCOW version 3 supports a 1.1 volume compatibility level.
+The QCOW version specifies to the qemu which qemu version the volume supports.
+The volume compatibility version is determined by the storage domain's version which the disk is created on.
+Storage domains with version lower than V4 support QCOW2 volumes, while V4 storage domains also support QCOW2v3.
```

Comment:
This is a series of disconnected sentences. Can we turn it into a single sentence? Something like
"The storage domain version dictated the maximum supported compat level of QCOW volumes. V4 domains support compat level of 1.1, while V3 domains (or older) support only compat level 0.10".